code/controllers/subsystems/job.dm

```diff
@@ -403,6 +403,17 @@
 
 	INVOKE_ASYNC(GLOBAL_PROC, .proc/show_location_blurb, H.client, 30)
 
+	if(SSticker.mode != "Extended" && joined_late)
```

Comment:
Not necessary to check if the mode is extended, because extended has zero things in its `antag_tags` list so it will just be skipped anyway.